tests/__init__.py

```diff
@@ -26,68 +31,109 @@
                  'subject5/session1',
                  'subject6/session1']
 
+# --------------------  HELPER CLASS --------------------
+
+
+class QuietStdOut:
+    """If verbose set to false, used to quiet tear_down table.delete prints"""
+    def __enter__(self):
+        self._original_stdout = sys.stdout
+        sys.stdout = open(os.devnull, 'w')
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        sys.stdout.close()
+        sys.stdout = self._original_stdout
+
+# ---------------------- FIXTURES ----------------------
 
-# ------------------- FIXTURES -------------------
 
 @pytest.fixture(autouse=True)
 def dj_config():
+    """ If dj_local_config exists, load"""
     if pathlib.Path('./dj_local_conf.json').exists():
         dj.config.load('./dj_local_conf.json')
     dj.config['safemode'] = False
-
     dj.config['custom'] = {
         'database.prefix': (os.environ.get('DATABASE_PREFIX')
                             or dj.config['custom']['database.prefix']),
-        'ephys_root_data_dir': (os.environ.get('EPHYS_ROOT_DATA_DIR')
+        'ephys_root_data_dir': (os.environ.get('EPHYS_ROOT_DATA_DIR').split(',')
                                 or dj.config['custom']['ephys_root_data_dir'])
     }
     return
 
 
 @pytest.fixture(autouse=True)
 def test_data(dj_config):
-    test_data_dir = pathlib.Path(dj.config['custom']['ephys_root_data_dir'])
-
-    test_data_exists = np.all([(test_data_dir / p).exists() for p in sessions_dirs])
+    """If data does not exist or partial data is present,
+    attempt download with DJArchive to the first listed root directory"""
+    test_data_dirs = []
+    test_data_exists = True
+    for p in sessions_dirs:
+        try:
+            test_data_dirs.append(find_full_path(get_ephys_root_data_dir(), p))
+        except FileNotFoundError:
+            test_data_exists = False   # If data not found
```

Comment:
```suggestion
    test_data_exists = True
    for p in sessions_dirs:
        try:
           find_full_path(get_ephys_root_data_dir(), p)
        except FileNotFoundError:
            test_data_exists = False   # If data not found
```

Reply:
Since `test_data_dirs` doesn't seem to be used in this function, perhaps we can just call `find_full_path`?